trillian/integration/hammer.go

```diff
@@ -449,42 +451,51 @@ func (s *hammerState) oneOp(ctx context.Context) error {
 	ep := s.cfg.EPBias.Choose()
 	glog.V(3).Infof("perform %s operation", ep)
 	status := http.StatusOK
+	defer func() {
+		s.stats.done(ep, status)
+		s.totalOps++
+	}()
+
 	var err error
-	switch ep {
-	case ctfe.AddChainName:
-		err = s.addMultiple(ctx, s.addChain)
-	case ctfe.AddPreChainName:
-		err = s.addMultiple(ctx, s.addPreChain)
-	case ctfe.GetSTHName:
-		err = s.getSTH(ctx)
-	case ctfe.GetSTHConsistencyName:
-		err = s.getSTHConsistency(ctx)
-	case ctfe.GetProofByHashName:
-		err = s.getProofByHash(ctx)
-	case ctfe.GetEntriesName:
-		err = s.getEntries(ctx)
-	case ctfe.GetRootsName:
-		err = s.getRoots(ctx)
-	case ctfe.GetEntryAndProofName:
-		status = http.StatusNotImplemented
-		glog.V(2).Infof("%s: hammering entrypoint %s not yet implemented", s.cfg.LogCfg.Prefix, ep)
-	default:
-		return fmt.Errorf("internal error: unknown entrypoint %s selected", ep)
-	}
+	for {
+		switch ep {
+		case ctfe.AddChainName:
+			err = s.addMultiple(ctx, s.addChain)
+		case ctfe.AddPreChainName:
+			err = s.addMultiple(ctx, s.addPreChain)
+		case ctfe.GetSTHName:
+			err = s.getSTH(ctx)
+		case ctfe.GetSTHConsistencyName:
+			err = s.getSTHConsistency(ctx)
+		case ctfe.GetProofByHashName:
+			err = s.getProofByHash(ctx)
+		case ctfe.GetEntriesName:
+			err = s.getEntries(ctx)
+		case ctfe.GetRootsName:
+			err = s.getRoots(ctx)
+		case ctfe.GetEntryAndProofName:
+			status = http.StatusNotImplemented
+			glog.V(2).Infof("%s: hammering entrypoint %s not yet implemented", s.cfg.LogCfg.Prefix, ep)
```

Comment:
Don't think so (I didn't change how this worked, just indented it), this looks like it's saying "not implemented", which should return - no amount of retying is going to fix that :)